tests/integration/states/test_file.py

```diff
@@ -2696,6 +2696,26 @@ def test_binary_contents(self):
             except OSError:
                 pass
 
+    def test_binary_contents_twice(self):
+        '''
+        This test ensures that after a binary file is created, salt can confirm
+        that the file is in the correct state.
+        '''
+        # Create a binary file
+        name = os.path.join(TMP, '1px.gif')
```

Comment:
FWIW this could be rewritten using `tempfile.NamedTemporaryFile`, to avoid the `try/except` to try and cleanup.